components/02-elements/button/Button.vue

```diff
@@ -1,34 +1,26 @@
+<template>
+  <component
+    :is="tag ? tag : 'button'"
+    :class="[tag && 'button',
+             customClass && customClass,
+             icon && 'button--icon',
+             secondary && 'button--secondary',
+             link && 'button--link',
+             fluid && 'button--fluid']">
```

Comment:
It should be in object, not array, will be way cleaner

Reply:
I'm not sure if that's that option (b/c we don't use JSX), but that's what we need to set to `false` https://prettier.io/docs/en/options.html#jsx-brackets